### ISSUE TITLE
Testing 1.16.2 - August 23rd Source

### ISSUE THREAD

**Issue by peekpeekpeekpeek** [sinkillerj/ProjectE#2055]
ProjectE 1.16.2 github branch
Forge 33.0.55
Minecraft 1.16.2 (From August 23rd, 16 days ago)

Since I wanted to play with 1.16.2, and saw there is a 1.16.2 branch, I downloaded the source from github, and built it. 

Amazingly it loaded up fine!
It appeared to work with jei-1.16.2-7.3.2.25.jar, all ProjectE items and their recipes listed.

I got pretty far, created a philosopher's stone, transmutation table, alchemical  chest, divining rod, MK1 collector, and matter condenser.
Philosopher's stone works fine.  Turns coal into alchemical coal, emeralds into diamonds and so on.
Transmutation table won't accept any items, so no EMC, no transmutation.
Divining rod counts blocks correctly, but EMC of blocks is always 1.
Alchemical chest has text labels overlapping chest contents (minor bug).
MK1 collector counts up EMC, but won't let you add items, so it just counts up.
Matter condenser doesn't appear to work (won't accept target item), and won't accept EMC from nearby collector.

**Comment by pupnewfster** [comment]
Thank you for your report, I don't really remember what state the 1.16 branch is at and what I only have WIP locally change wise. I do recall that at one point nothing had EMC values and this has been fixed (at least locally) since then, but that I have other WIP changes that are not quite ready to be committed. My best guess is all the issues you are running into are due to this so I am closing this for now as when I had spent a little time earlier today with my local branch continuing to work on things I was not running into any of those issues beyond some GUI things which I have TODOs for in the code already to keep track of them.
